### DISCUSSION TITLE
How to select a LUT

### DISCUSSION THREAD

**Discussion by markspen** [latenitefilms/BRAWToolbox#34] [Support] [answered]
There's an option for having the LUT be embedded, sidecar or none, but I don't see where you are able to choose the LUT you want to apply in the extension, AND you can't see or apply a camera LUT after import since the dialog is not available. What is the correct workflow for applying LUTs to BRAW footage?

**Comment by latenitefilms** [comment, marked as answer]
See: https://brawtoolbox.io/luts/
